game_eggs/tycoon_games/openrct2/README.md

```diff
@@ -0,0 +1,88 @@
+        </td>
+        <td align="center">
+            <a href="https://github.com/janisozaur">
+                <img src="https://avatars.githubusercontent.com/u/550290" width="50px;" alt=""/><br /><sub><b>janisozaur</b></sub>
+            </a>
+            <br />
+            <a href="https://github.com/OpenRCT2/OpenRCT2/commits?author=janisozaur" title="Codes">💻</a>
+            <a href="https://github.com/OpenRCT2/OpenRCT2/commits?author=janisozaur" title="Contributor">💡</a>
+        </td>
+    </tr>
+</table>
+<!-- markdownlint-enable -->
+<!-- prettier-ignore-end -->
+
+___
+### Game Description
+From OpenRCT2's [Website](https://openrct2.org/):
+> OpenRCT2 is an open-source re-implementation of RollerCoaster Tycoon 2 (RCT2), expanding the game with new features, fixing bugs and raising game limits. The gameplay revolves around building and maintaining an amusement park containing attractions, shops and facilities. The player must try to make a profit and maintain a good park reputation whilst keeping the guests happy. OpenRCT2 allows for both scenario and sandbox play. Scenarios require the player to complete a certain objective in a set time limit whilst sandbox allows the player to build a more flexible park with optionally no restrictions or finance.
+___
+### Server Ports
+| Port | Default (TCP) |
+|---------|---------|
+| Game | 11753 |
+
+___
+### Installation/System Requirements
+
+|  | Minimum |
+|---------|---------|
+| Processor | Recent x86/64 (AMD/Intel) processor. No ARM support. |
+| RAM | 256-512 MiB (depends on # of clients & park size) |
+| Storage | 80 MB |
+| RCT2 Files | **Not required** |
+
+___
+### Save Files
+Save files and autosaves are located in this directory:
+```
+/home/container/ServerData/save/
+```
+If the "Load Latest Autosave" startup variable is enabled, and the `autosave/` directory exists, the server will load the latest autosave instead of the specified Save File.
+___
+### Console Commands
+As of OpenRCT2 v0.3.5.1, entering commands via Pterodactyl do nothing. Hopefully this will be resolved in a future OpenRCT2 update.
+___
+### Becoming an Admin on the Server
+Unfortunately, because console commands currently do not register, becoming the first admin on the server takes some steps, listed below:
+1. Ensure the "Log Server Actions" startup variable is set to "true".
+2. Start the server and join it with the client you want to make an admin.
+3. Open `ServerData/serverlogs/<Server Name>/<Latest Date>.txt` and copy the SHA1 hash next to your client's username (note the username as well).
+4. Then, open `ServerData/users.json`. Paste the hash into the "hash" value, and change the "name" value to your client's username.
+5. Reboot the server, re-join it, and you should be an admin!
+
+After becoming an admin, you can use the in-game users management window to add additional admins (or groups) more easily if you wish.
```

Comment:
There is an issue already filed to ease this, please link it here

Reply:
Software-Noob:
> Pterodactyl does not support interactive programs at this time.

Without direct console input on the host, I don't see how this issue can be eased any further than having a client obtain their key and pasting it into `users.json` in some fashion. Unless, you are referring to my feature request OpenRCT2/OpenRCT2#16418, I don't know what issue you are referring to.

Regardless, game server issues and feature requests are typically not relevant to Pterodactyl Egg READMEs, as the README is only supposed to serve info for the egg itself. Eggs (once merged) don't get updated as often as the game server typically does, so leaving any current-day issues linked in the README may quickly fall into obscurity. However, I will add it if @Software-Noob thinks it should be added.

Reply:
However, I may link this issue pterodactyl/panel#3712 since it is relative to Pterodactyl.

Reply:
Yes, I meant 16418. I will leave it for you to decide then

Reply:
I will submit a new PR to update this section of the README in the future if/when it becomes any easier to add the first admin to the server.